pf4j/src/main/java/ro/fortsoft/pf4j/DefaultExtensionFinder.java

```diff
@@ -51,10 +51,10 @@ public DefaultExtensionFinder(PluginManager pluginManager, ExtensionFactory exte
         }
 
 		log.debug("Finding extensions for extension point '{}'", type.getName());
-        readIndexFiles();
+        Map<String, Set<String>> cachedEntries = readIndexFiles();
```

Comment:
It was to make it easy to test but if you have another suggestion I can revert this.